tools/roslaunch/src/roslaunch/rlutil.py

```diff
@@ -90,7 +89,7 @@ def resolve_launch_arguments(args):
     # try to resolve launch file in package first
     if len(args) >= 2:
         try:
-            resolved = roslaunch.resource_cache.find_resource(args[0], args[1])
+            resolved = roslib.packages.find_resource(args[0], args[1])
```

Comment:
Actually I don't fully understand about rospack instance. I just wonder we can pass `rospack` instance to `find_resource` function which has cached `source _path_to_package` as previous implementation.